modules/infra/src/main/scala/scaladex/infra/sql/ArtifactTable.scala

```diff
@@ -114,26 +118,16 @@ object ArtifactTable {
       groupBy = projectReferenceFields
     )
 
-  def selectLatestArtifacts(stableOnly: Boolean): Query[Project.Reference, Artifact] =
-    selectRequest1(latestDateTable(stableOnly), fields.map(c => s"a.$c"))
-
-  // the latest release date of all artifact IDs
-  private def latestDateTable(stableOnly: Boolean): String =
-    s"($table a " +
-      s"INNER JOIN (${selectLatestDate(stableOnly).sql}) d " +
-      s"ON a.group_id=d.group_id " +
-      s"AND a.artifact_id=d.artifact_id " +
-      s"AND a.release_date=d.release_date)"
-
-  private def selectLatestDate(
-      stableOnly: Boolean
-  ): Query[Project.Reference, (Artifact.GroupId, String, Instant)] = {
-    val isReleaseFilters = if (stableOnly) Seq("is_semantic='true'", "is_prerelease='false'") else Seq.empty
-    selectRequest1(
+  def selectLatestArtifacts: Query[Project.Reference, Artifact] =
+    selectRequest1(table, fields, where = Seq("organization=?", "repository=?", "is_latest_version=true"))
+
+  def setLatestVersion: Update[Artifact.MavenReference] =
+    updateRequest0(table, set = Seq("is_latest_version=true"), where = Seq("group_id=?", "artifact_id=?", "version=?"))
+
+  def unsetOthersLatestVersion: Update[Artifact.MavenReference] =
+    updateRequest0(
       table,
-      Seq("group_id", "artifact_id", "MAX(release_date) as release_date"),
-      where = Seq("release_date IS NOT NULL") ++ isReleaseFilters ++ Seq("organization=?", "repository=?"),
-      groupBy = Seq("group_id", "artifact_id")
+      set = Seq("is_latest_version=false"),
+      where = Seq("group_id=?", "artifact_id=?", "version<>?")
     )
-  }
```

Comment:
Just checking - it looks like the old code would check `"is_semantic='true'", "is_prerelease='false'"` if `stableOnly` was true - ie there could be more than one 'latest' for a set of artifacts, depending on whether you only wanted to see stable artifacts or not. Is that logic removed now? I think personally I'd prefer that the 'latest' be restricted to the latest _stable_ version?

I'm not sure where this value is surfaced but I would guess it's here, near the top of the [project page](https://index.scala-lang.org/scanamo/scanamo) by the repo name: 

<img width="1450" alt="scaladex" src="https://github.com/user-attachments/assets/b8cb0d8a-bf3d-4768-a253-230d34d3f741">

...would that `2.0.0` now become the very long pre-release version number (`3.0.0-PREVIEW.add-transact-to-expose-TransactionCanceledException.2024-08-10T0658.6eeb6522`)?

Reply:
There is only one set of latest artifacts at a time, but it can be stable or maybe not stable depending on the project settings. The `stableOnly ` setting is used when we compute the `is_latest_version`, and if the project setting changes we re-compute it.

Also I already deployed this PR, so what you see on the website is the current PR. You can try to log in, go to the project settings, disable the `Stable Only` filter and you should see the new latest version becomes the very long pre-release version.

Reply:
Thanks @adpi2 that makes sense! The only thing is that, even though I'm logged in on Scaladex, I don't think I've ever been able to access my projects settings - where is that functionality?

Here are a couple of projects that I think I should be able to administer:

<img width="1445" alt="image" src="https://github.com/user-attachments/assets/b8fe4c30-ceed-4299-af72-5f002a8c2711">
<img width="1445" alt="image" src="https://github.com/user-attachments/assets/511aed37-b52f-44c5-a2c8-3b7e3dabe412">

If I click on `My projects` from the dropdown, I only see this:

<img width="1445" alt="image" src="https://github.com/user-attachments/assets/1f4e02ac-0c3f-4f64-97ac-74ad31918fc9">

I don't actually _want_ to be able to disable the `Stable only` flag, I'm just curious how I would do it!?

Reply:
Ah yes, that's because it takes a very long time to get the list of projects, and organizations of a user using the Github API, and so we do it asynchronously. Usually it takes about 30 seconds. But yesterday I had the impression that it was taking forever, or that it was failing. Maybe come back to your project in a few minutes.

Also if you log out it will reset the timer, so after logging in again you'll have to wait again.

Reply:
@rtyley Are you able to access you project settings? If not could you open a new issue.

Reply:
> Are you able to access you project settings?

Only on repos which I directly own, not organisation-held ones! I've opened https://github.com/scalacenter/scaladex/issues/1445 to track that.

Reply:
Thanks!